beacon_node/lighthouse_network/src/rpc/methods.rs

```diff
@@ -30,14 +30,22 @@ pub const MAX_ERROR_LEN: u64 = 256;
 pub struct ErrorType(pub VariableList<u8, MaxErrorLen>);
 
 impl From<String> for ErrorType {
+    // This will panic if `string.as_bytes()` exceeds `MaxErrorLen`.
     fn from(s: String) -> Self {
-        Self(VariableList::from(s.as_bytes().to_vec()))
+        Self(
+            VariableList::try_from(s.as_bytes().to_vec())
+                .expect("length should not exceed MaxErrorLen"),
```

Comment:
Maybe we should keep the truncating behaviour here. It would be bad to crash LH due to an error being too long.

We don't necessarily need a method on `VariableList`, we could just do `.truncate()` on the bytes before passing them to `VariableList::try_from`.